components/raftstore/src/store/txn_ext.rs

```diff
@@ -47,7 +47,47 @@ impl fmt::Debug for TxnExt {
 
 /// Pessimistic locks of a region peer.
 pub struct PeerPessimisticLocks {
-    pub map: HashMap<Key, PessimisticLock>,
+    /// The table that stores pessimistic locks.
```

Comment:
Should this part be added in the rfc document or the test plan, it will be helpful to the test participants and to verify if all the situations are covered.

Reply:
Ah, yes. There are quite some changes to the original design.